### DISCUSSION TITLE
Filter unrealistic Values in HomeAssistant

### DISCUSSION THREAD

**Discussion by Alex-joomla** [jomjol/AI-on-the-edge-device#1744] [Q&A]
As many of you running the AI Meter as part of a Homeassistant installation i have a question about data validity. I had quite a few problems with my meter [(more)](https://github.com/jomjol/AI-on-the-edge-device/discussions/1437) first the installation itself, most of the time with the alignment and sometimes with the recognition and I don't get it any better. 
Spent hours but no success. Nevertheless, in the majority of cases I get a correct value via mqtt in my HA. But If the recognition is incorrect there are sometimes complete crazy measurings, leading to faulty values in the database. (Sky high values, as you can see in the graphic). 
Is there a measure to avoid these values from being collected in the database. 
Played around with the outliers filter but didn't get satisfying results. The integrated (value too high..) didn't work and I want a 2nd safety measure.

Any Ideas?
<img width="647" alt="Bildschirmfoto 2023-01-02 um 12 55 33" src="https://user-images.githubusercontent.com/32268233/210230947-6103e886-59aa-4478-b67b-976afa8e5c77.png">
<img width="796" alt="Bildschirmfoto 2023-01-02 um 13 25 46" src="https://user-images.githubusercontent.com/32268233/210231050-3846bfca-3cab-4fbf-96bc-deb9af6c4f6b.png">

**Comment by ghost** [comment]
You could collect the images with https://github.com/jomjol/AI-on-the-edge-device/wiki/Learn-models-with-your-own-images 
Wich dig-model you use? You could try another one.

**Reply by Alex-joomla** [reply]
Tried every Model, even sent Training Data in but yes, maybe I will try to take new training samples

**Reply by ghost** [reply]
Oh, I remember. Here only helping more images, I think. 

If the bar above the two is a tape, you can remove it at that point and use the last digit. It is not necessary here that the post has exactly the same height as the previous ones.
